corehq/apps/hqadmin/templates/hqadmin/bootstrap3/hqadmin_base_filters.html

```diff
@@ -0,0 +1,36 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div class="hq-accordion-control">
```

Comment:
again, this is from the copied part of the file. It might be good to review commit-by-commit to see what I updated? This isn't shown on this page, so I didn't work on updating the styles yet. I'm saving that for a page that actually uses filters...keeping the changes here as basic as possible